start-site/src/main/resources/application.yml

```diff
@@ -436,11 +436,20 @@ initializr:
               description: Authenticating a User with LDAP
             - rel: reference
               href: https://docs.spring.io/spring-boot/docs/{bootVersion}/reference/htmlsingle/#web.security
+        - name: Spring Authorization Server
+          id: oauth2-authorization-server
+          description: Spring Boot integration for Spring Security's authorization server features.
+          facets:
+            - native
+          links:
+            - rel: reference
+              href: https://docs.spring.io/spring-boot/docs/{bootVersion}/reference/htmlsingle/#web.security.oauth2.authorization-server
         - name: OAuth2 Client
           id: oauth2-client
           description: Spring Boot integration for Spring Security's OAuth2/OpenID Connect client features.
           facets:
             - native
+          compatibilityRange: 3.1.0-RC1
```

Comment:
It looks like the compatibility range was added on the wrong entry? Also please quote the value for consistency (see Andy's comment).

Reply:
@snicoll I accidentally added it to the wrong dependency😅 Thank you for your help!

Reply:
Sure thing, thanks.